packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -37,6 +39,7 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
   pagination,
   tableApi,
   onChangePage,
+  noDataMessageKey,
```

Comment:
forcing the use of message keys rather than strings in the hope that it will prevent people being lazy in the future and not using translations. possibly over cautious?